### PR TITLE
[CS0618] `InternalFlowOperations` - Warning Disable

### PR DESCRIPTION
**Pull request by eaba**
## Checklist

For significant changes, please ensure that the following have been completed (delete if not relevant):

* [x] This change follows the [Akka.NET API Compatibility Guidelines](https://getakka.net/community/contributing/api-changes-compatibility.html).
* [x] This change follows the [Akka.NET Wire Compatibility Guidelines](https://getakka.net/community/contributing/wire-compatibility.html).

### CODE REVIEW

File: src/core/Akka.Streams/Dsl/Internal/InternalFlowOperations.cs [hunk 1]
```diff
@@ -1284,7 +1284,9 @@ internal static class InternalFlowOperations
         /// <param name="stageFactory">TBD</param>
         /// <returns>TBD</returns>
         public static IFlow<TOut, TMat> Transform<TIn, TOut, TMat>(this IFlow<TIn, TMat> flow,
+#pragma warning disable CS0618 // Type or member is obsolete
             Func<IStage<TIn, TOut>> stageFactory)
+#pragma warning restore CS0618 // Type or member is obsolete
```
Comment (eaba) [inline src/core/Akka.Streams/Dsl/Internal/InternalFlowOperations.cs:1289]: `IStage` Obsolete

**Review by eaba — COMMENTED** [review]
Use `GraphStage` instead because `IStage` it is `Obsolete` however `IStage` it is an `Interface`

File: src/core/Akka.Streams/Dsl/Internal/InternalFlowOperations.cs (hunk 1 above)
Comment (Aaronontheweb) [reply src/core/Akka.Streams/Dsl/Internal/InternalFlowOperations.cs:1289]: I'm sorry, I don't understand - why is disabling the warning the right fix here versus marking this method as obsolete itself?

Comment (eaba) [reply src/core/Akka.Streams/Dsl/Internal/InternalFlowOperations.cs:1289]: https://github.com/akkadotnet/akka.net/blob/2b7fef5f5d7cdf8c0f20756f40fbc6916c12822b/src/core/Akka.Streams/Stage/Stage.cs#L34-L35

`IStage<TIn, TOut>` it is an `Interface`

**Comment by HamzaAmjad-RG** [comment]
Hey, Can you help me with the PR, I changed a word in Documentation here: https://getakka.net/articles/actors/receive-actor-api.html   and two DocFx tests are failing.

**Comment by Aaronontheweb** [comment]
> Hey, Can you help me with the PR, I changed a word in Documentation here: https://getakka.net/articles/actors/receive-actor-api.html and two DocFx tests are failing.

Which PR?

**Pull request merged**